app/uk/gov/hmrc/traderservices/controllers/FormFieldMappings.scala

```diff
@@ -149,7 +149,7 @@ object FormFieldMappings {
 
   val importHasALVSMapping: Mapping[Boolean] = booleanMapping("importHasALVS", "yes", "no")
 
-  val allowedSpecialNameCharacterSet = Set(' ', '/', '\\', '_', '-', '&', '+', '\'', '.')
+  val allowedSpecialNameCharacterSet = Set(' ', '-', '&', '\'', '.')
```

Comment:
Even though the content for this message doesn't include some of the symbols we allowed I think we're alright to leave them in. This is Irene's comment on the spreadsheet 
<img width="323" alt="image" src="https://user-images.githubusercontent.com/55190161/96860208-62ac4000-145a-11eb-936b-6d7909b08851.png">